masterlist.yaml

```diff
@@ -7344,12 +7344,13 @@ plugins:
         name: 'Qwinn''s Refined FNV Redesigned 3'
       - link: 'https://www.nexusmods.com/newvegas/mods/72951/'
         name: 'NVR3R - New Vegas Redesigned 3 Revised'
-    after: [ 'Project Beauty.esm' ]
+    after:
+      - 'Project Beauty.esm'
+      - 'Two Wastelands Redesigned.esm'
     inc:
       - 'New Vegas Redesigned.esm'
       - 'New Vegas Redesigned II.esm'
       - 'TTW Redesigned.esm'
-      - 'Two Wastelands Redesigned.esm'
```

Comment:
Removing the inc would be fine, it's possible to patch the two manually.
I'd be reluctant to add the load after rule though, as it could come down to user preference.

Reply:
Hmm...that is a fair point. Okay, removing the incompatibility only it is!

Reply:
I generally avoid changing order between NPC appearance mods.
Unless there's a good reason, as it's potentially a can of worms.